### ISSUE TITLE
Could you please share the hyperparameters?

### ISSUE THREAD

**Issue by HVQuan02** [Alibaba-MIIL/PETA#15]
I trained your model with hyperparameters as described in STAM16: album batch size 32, learning rate 1e-5, adam optimizer with weight decay 1e-3, 100 max epoch with 10 linear warmup, cosine annealing scheduler, your defined asymmetric loss and additional ema model, it took me 7 hours to train. However, the map result did not converge to 90% but only 30%, what;s wrong ?! I use pytorch average_precision_score for map metric bc your validate function gave strange result (map value > 1 million), and val set is 300 albums. Thanks for reading!

**Comment by zhemulin** [comment]
Could you share the dataset since it is not possible to download it anymore? Thank you!@HVQuan02

**Comment by zhemulin** [comment]
The validate function must make sure the val_set covers all types of photo albums.Otherwise,ap[k] = average_precision(scores, targets),when ap.mean(),the albums not in val_set will get a zero_ap.So  creating a mapping between class_map = {0: "Birthday", 1: "Graduation", 2: "PersonalSports", 3: "Show", 4: "ThemePark"} and classes_list.And the order of different classes maybe must  be sorted alphabetically.
